pkg/virtualizers/firecracker/virtualizer_linux.go

```diff
@@ -88,6 +89,9 @@ func SetupBridgeAndDHCPServer() error {
 	go func() {
 		http.ListenAndServe(":7476", nil)
 	}()
+	tempLog := elog.CLI{}
```

Comment:
Why create a logger, shouldn't fmt.Printf() be good enough?

Reply:
Decided to use our library over the standard. I guess printf is technically just better in general